config/flags_node.go

```diff
@@ -270,6 +270,13 @@ var (
 		Name:  "resident-country",
 		Usage: "set resident country. If not set initially a default country will be resolved.",
 	}
+
+	// FlagNATStatusPollInterval nat status poll interval in seconds
+	FlagNATStatusPollInterval = cli.DurationFlag{
+		Name:  "nat-status.poll",
+		Usage: `NAT status poll interval in seconds. Lowest interval of 1 minute is allowed.`,
+		Value: 5 * 60 * time.Second,
```

Comment:
Isn't this just `5 * time.Minute`? I mean for time.Duration [it doesnt matter](https://play.golang.org/p/RCitzq3VLmt) how you give the time.
Also the `Usage` comment is a little weird. Why in seconds? I think it'd actually work if given like `3m43s` 🤷🏻